app/assets/stylesheets/forever_style_guide/globals/_effects.scss

```diff
@@ -74,6 +74,12 @@ $color-card-bg: color('white');
 }
 .card {
   @extend %card-style;
+
+  @each $id, $color in $colors {
```

Comment:
Rather than duplicating this loop, we could just include this in colors.scss

```
@each $id, $color in $colors {
  .color-#{$id} {
    color: #{$color};
  }
  .color_block-#{$id},
  .card.color_block-#{$id} {
    background-color: #{$color};
  }
  .color_block-trans-#{$id} {
    background-color: #{$color};
    background-color: rgba($color, 0.9);
  }
}
```

It's less clearly associated w/ the .card class when reading the code, but I think in practice, adding a color_block-color class to a card element and having that color override would be the expected behavior.

Thoughts?